msctl

```diff
@@ -1209,9 +1220,15 @@ start() {
     exit 1
   fi
   # Start the server.
-  sh -c "tail -f --pid=\$$ '$WORLD_DIR/console.in' | {
-    $SERVER_COMMAND mscs-world='$1' > '$WORLD_DIR/console.out' 2>&1; kill \$$;
-    }" &
+  if [ $RUN_IN_FOREGROUND -eq 1 ]; then
+    sh -c "tail -f --pid=\$$ '$WORLD_DIR/console.in' | {
+      $SERVER_COMMAND mscs-world='$1' > '$WORLD_DIR/console.out' 2>&1; kill \$$;
+      }"
+  else
+    sh -c "tail -f --pid=\$$ '$WORLD_DIR/console.in' | {
+      $SERVER_COMMAND mscs-world='$1' > '$WORLD_DIR/console.out' 2>&1; kill \$$;
+      }" &
+  fi
   # Verify the server is running.
```

Comment:
How does this function with multiple worlds?  It seems like this would get stuck starting the first world in the list and wouldn't try starting any others until that first one exited.